src/Modules/Backend/templates/base/formTheme.html.twig

```diff
@@ -642,3 +642,12 @@
     {{ form_widget(form.children|last) }}
   </div>
 {% endblock %}
+
+{% block toggle_password_widget %}
+  <div class="input-group mb-3">
+    {{ form_widget(form) }}
+    <button class="btn btn-default" type="button" data-role="toggle-visibility" data-target="{{ form.vars.id }}">
```

Comment:
Wouldn't it make more sense to set the data-role on the input group? So the JS can be more scoped on the full element instead of only the button.

Reply:
Fixed 2e56f912c